src/rxjs/atomWithObservable.ts

```diff
@@ -1,23 +1,51 @@
-import { Observable, Subject } from 'rxjs'
-import { first } from 'rxjs/operators'
 import { atom } from 'jotai'
 import type { Atom, WritableAtom, Getter } from 'jotai'
 
+const observableSymbol: typeof Symbol.observable =
+  typeof Symbol === 'function'
+    ? Symbol.observable || ((Symbol as any).observable = Symbol('observable'))
```

Comment:
After looking at the code, it's basically used only in L46-L47.
So, this could just be:
```js
const observableSymbol = Symbol.observable || undefined
```
(meaning we don't need this)

Or, is there any specific reason we want to assign `Symbol.observable` with a unique symbol?

Reply:
I think it's an attempt to polyfill the `Symbol.observable`: https://github.com/benlesh/symbol-observable/blob/master/es/ponyfill.js

I'm not sure if it's the correct implementation.

Reply:
Thanks. In such a case, `const observableSymbol = Symbol.observable` should be fine for us, and if really required, people should polyfill (not ponyfill) the symbol in their app code.